### ISSUE TITLE
Buttons: white text on cyan background fails the simplest contrast test (WCAG:AA) 

### ISSUE THREAD

**Issue by tdupre** [PrestaShop/prestashop-ui-kit#155]
# Readability and accessibility 

Readability and accessibility are paramount these days and companies get fined for not complying with accessibility basic rules ([in the US](https://www.govloop.com/community/blog/3-great-risks-ada-compliance-failures/) and [soon in Europe](https://siteimprove.com/en/accessibility/eu-web-accessibility-directive/))

# Acccessibility issue
The buttons used in PrestaShop UI kit fail the simplest contrast test (WCAG:AA) Foreground:#FFFFFF background:#2eb9d7

https://color.a11y.com/ContrastPair/?bgcolor=2eb9d7&fgcolor=ffffff

![image](https://user-images.githubusercontent.com/18527605/116533328-88bb0d00-a8e1-11eb-9330-165ae02a5bd5.png)

# Fix proposition
One way to fix this would be to use the purple that's available in PrestaShop's colors #442CC7 ensuring a good contrast between the text and the background

https://color.a11y.com/ContrastPair/?bgcolor=442CC7&fgcolor=ffffff

![image](https://user-images.githubusercontent.com/18527605/116534218-8b6a3200-a8e2-11eb-829d-d8bea5c28029.png)

**Comment by ldeprestashop** [comment]
@PrestaShop/prestashop-core-developers is it just a CSS property ?

**Comment by matks** [comment]
Hello @ldeprestashop yes, the settings being discussed here are related to CSS.

However changing CSS values is not a decision we can make alone. These colors have been chosen to follow PrestaShop brand identity.

**Comment by ldeprestashop** [comment]
OK thanks :) 

I will meet Thibaut this afternoon to discuss about that.

**Comment by prestascott** [comment]
Hello everyone,

Thanks @tdupre for this issue 🙂

We know that the current color `#2eb9d7` used as primary  in the back office doesn't even pass the WCAG AA test and that's terrible... I already discussed with Tristan about this accessibility issue and we did some color tests to see the look and feel of the back office. 

In fact, we did a color test with the `#442cc7` but it's too dark. Also, the [Google tools for picking colors](https://material.io/design/color/the-color-system.html#tools-for-picking-colors) defined it as 700 which is pretty gloomy. 

See the screenshot:
![image](https://user-images.githubusercontent.com/54888056/116562688-ac8f4a80-a903-11eb-9dd9-ac4316c02d8f.png)

We didn't test the `#6039d6` (500) as default because it passes the WCAG AAA but not the WCAG AA ❎

See the screenshot from [this website](https://color.a11y.com/ContrastPair/?bgcolor=6039D6&fgcolor=ffffff):
![Capture d’écran 2021-04-29 à 15 09 14](https://user-images.githubusercontent.com/54888056/116563546-68507a00-a904-11eb-94ae-691e4da92a7d.png)

Se we tried `#5434d0` (600) as default because it passes the WCAG AAA and the WCAG AA ✅

See the screenshot from [this website](https://color.a11y.com/ContrastPair/?bgcolor=5434d0&fgcolor=ffffff):
![Capture d’écran 2021-04-29 à 15 09 27](https://user-images.githubusercontent.com/54888056/116563631-7dc5a400-a904-11eb-8989-03a6369f2a23.png)

You can see the color test on the [Figma file](https://www.figma.com/file/jMR2PSSVyqDRQtHJLRjmYG/UI-Kit-Core?node-id=510%3A14624).

Let me know what do you think ✌️

**Comment by gsauvanon-presta** [comment]
Hi there, I'm Guillaume, product designer of PS Metrics, is that could help, you can see in this [file](https://www.figma.com/file/OgUXcAAdzk0yBUQ5zm4CMz/2.0-Page-d%C3%A9di%C3%A9e?node-id=1%3A66) our interface with buttons as purple color @prestascott we have a purple a little darker than your proposal, tell me what you think about this one : 34219E (who is the same as the logo of Prestashop), thanks!

**Comment by prestascott** [comment]
Hey @gsauvanon-presta 👋🏻

Thanks for your response. The purple color used `#34219E` passes WCAG AA and AAA but it's really way too dark in my opinion. Plus, the Google tools considers it as 900-level. 

I need to use a 600-level maximum in order to user higher levels for events such as Hover, Pressed. If the primary color is too dark, it will be difficult to differentiate Hover status than Default status 😅

**Comment by gsauvanon-presta** [comment]
hey @prestascott thanks for your advice, sounds great to use #5434d0, I understand that our purple is to dark :smiling_imp: 
We'll change as soon as it will be official :)
